core/che-core-api-dto/src/test/java/org/eclipse/che/dto/definitions/DTOHierarchy.java

```diff
@@ -46,5 +46,18 @@
         void setParentField(String parentField);
 
         ChildDto withParentField(String parentField);
+
```

Comment:
Gson fails if a class has a shadowed field (a field with the same name as another field in a base class). To avoid this shadowed fields are omitted from generated classes. @skabashnyuk